src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/params.py

```diff
@@ -750,5 +750,9 @@ def _configure_security_policy_storage_params(cmd):
                help='Name or ID of the subnet that allows access to an Azure Sql Server. '
                'If subnet name is provided, --vnet-name must be provided.')
 
+    c.argument('ignore_missing_vnet_service_endpoint',
+               options_list=('--ignore-missing-vnet-service-endpoint', '-i'), action='store_true',
```

Comment:
Would be nice to abbreviate a little, maybe `--ignore-missing-endpoint`? Just a suggestion since you already have the `-i` shortcut.